susemanager-utils/dracut-mgrbootstrap/mgrbootstrap/mgrbootstrap-root.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+root=saltboot
+rootok=1
+
+# If we don't have a server, we need dhcp
+if [ -z "$server" ] ; then
+    DHCPORSERVER="1"
+fi;
+
+export DISABLE_UNIQUE_SUFFIX=$(getarg DISABLE_UNIQUE_SUFFIX=)
+export kiwidebug=$(getarg kiwidebug=)
```

Comment:
These two are not needed for your usecase.

Reply:
I wonder if I need mgrbootstrap-root.sh at all?

Reply:
I think you don't need that at all. Just rename the `preregister` to `mgrbootstrap.sh ` so it is correctly installed as pre-mount hook.

Reply:
I need mgrbootstrap-root.sh at least to overwrite the empty `root=` .

Reply:
Ah sorry, I misread. I though we were talking about `mgrbootstrap.sh`. My mistake.

Reply:
minimized